server/src/server.ts

```diff
@@ -171,22 +107,46 @@ connection.onInitialized(async () => {
 
 documents.onDidSave(async (change) => {
     connection.console.log('onDidSave event');
+
     await validateDocument(change.document);
 });
 
 documents.onDidOpen(async (change) => {
     connection.console.log('onDidOpen event');
+
     await validateDocument(change.document);
 });
 
+connection.onHover(({ textDocument, position }: HoverParams) => {
+    connection.console.log('onHover event');
+
+    const info = documentInfos.infoAt(textDocument.uri, position);
+    if (info) {
+        const content: MarkupContent = { kind: MarkupKind.PlainText, value: info.type };
+        const hover: Hover = { contents: content };
+
+        return hover;
+    }
+
+    return null;
+});
```

Comment:
```suggestion
connection.onHover(({ textDocument, position }: HoverParams): Hover => {
    connection.console.log('onHover event');

    const info = documentInfos.infoAt(textDocument.uri, position);
    if (info) {
        const content: MarkupContent = { kind: MarkupKind.PlainText, value: info.type };
        return { contents: content };
    }

    return null;
});
```
Seems a little bit cleaner but up to you